### PR TITLE
Use ctapipe restructured containers

### PR DESCRIPTION
**Pull request by sizun**
This PR includes the minimal changes to use NectarCAMEventSource after the ctapipe containers were restructured; refs #11.

### CODE REVIEW

File: ctapipe_io_nectarcam/__init__.py [hunk 1]
```diff
@@ -123,6 +123,19 @@ def prepare_subarray_info(self):
...
+    def is_simulation(self):
+        return False
+
+    @property
+    def datalevels(self):
+        return (DataLevel.R0, DataLevel.R1)
+
+    @property
+    def obs_id(self):
+        return -1  # TODO
```
Comment (MaxNoe) [inline ctapipe_io_nectarcam/__init__.py:136]: You runs don't contain a run number / observation id?

File: ctapipe_io_nectarcam/tests/test_nectarcameventsource.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def test_loop_over_events():
         for telid in event.r0.tels_with_data:
             assert event.index.event_id == FIRST_EVENT_NUMBER_IN_FILE + i
             n_gain = 2
-            n_camera_pixels = event.inst.subarray.tels[telid].camera.n_pixels
+            n_camera_pixels = inputfile_reader.subarray().tels[0].camera.n_pixels
```
Comment (MaxNoe) [inline ctapipe_io_nectarcam/tests/test_nectarcameventsource.py:21]: subarray should be a property, so this should just read `inputfile_reader.subarray.tels[0].camera.n_pixels`. 

Also: in ctapipe the tel ids start with 1 (also simtel)

File: ctapipe_io_nectarcam/__init__.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ def __init__(self, **kwargs):
         self.data = None
         self.log.info("Read {} input files".format(self.multi_file.num_inputs()))
 
-
+    @property
     def subarray(self):
         return self.prepare_subarray_info()
```
Comment (MaxNoe) [inline ctapipe_io_nectarcam/__init__.py:86]: Processing will very often access this, you probably should do something like

```
self._subarray = self.prepare_subarray_info()
```
 in the `__init__` method and here just `return self._subarray`.

File: ctapipe_io_nectarcam/tests/test_nectarcameventsource.py (hunk 2 above)
Comment (sizun) [reply ctapipe_io_nectarcam/tests/test_nectarcameventsource.py]: > Also: in ctapipe the tel ids start with 1 (also simtel)

When exactly to change tel_id will need to be discussed. All NectarCAM data produced so far have tel_id=0. Is there a table mapping telescopes to their ID?

**Comment by sizun** [comment]
@MaxNoe or @FrancaCassol , could you please merge this PR?

**Pull request merged**